cmd/troubleshoot/cli/interactive_results.go

```diff
@@ -142,11 +128,143 @@ func drawFooter() {
 }
 
+func drawAnalyzersTable(analyzeResults []*analyzerunner.AnalyzeResult) {
+	termWidth, termHeight := ui.TerminalDimensions()
+
+	table.SetRect(0, 3, termWidth/2, termHeight-6)
+	table.FillRow = true
+	table.Border = true
+	table.Rows = [][]string{}
+	table.ColumnWidths = []int{termWidth}
+
+	for i, analyzeResult := range analyzeResults {
+		title := analyzeResult.Title
+		if analyzeResult.IsPass {
+			title = fmt.Sprintf("✔  %s", title)
+		} else if analyzeResult.IsWarn {
+			title = fmt.Sprintf("⚠️  %s", title)
+		} else if analyzeResult.IsFail {
+			title = fmt.Sprintf("✘  %s", title)
+		}
+		table.Rows = append(table.Rows, []string{
+			title,
+		})
+
+		if analyzeResult.IsPass {
+			if i == selectedResult {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorGreen, ui.ColorClear, ui.ModifierReverse)
+			} else {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorGreen, ui.ColorClear)
+			}
+		} else if analyzeResult.IsWarn {
+			if i == selectedResult {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorYellow, ui.ColorClear, ui.ModifierReverse)
+			} else {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorYellow, ui.ColorClear)
+			}
+		} else if analyzeResult.IsFail {
+			if i == selectedResult {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorRed, ui.ColorClear, ui.ModifierReverse)
+			} else {
+				table.RowStyles[i] = ui.NewStyle(ui.ColorRed, ui.ColorClear)
+			}
+		}
+	}
+
+	ui.Render(table)
+}
+
+func drawDetails(analysisResult *analyzerunner.AnalyzeResult) {
+	termWidth, _ := ui.TerminalDimensions()
+
+	currentTop := 4
+	title := widgets.NewParagraph()
+	title.Text = analysisResult.Title
+	title.Border = false
+	if analysisResult.IsPass {
+		title.TextStyle = ui.NewStyle(ui.ColorGreen, ui.ColorClear, ui.ModifierBold)
+	} else if analysisResult.IsWarn {
+		title.TextStyle = ui.NewStyle(ui.ColorYellow, ui.ColorClear, ui.ModifierBold)
+	} else if analysisResult.IsFail {
+		title.TextStyle = ui.NewStyle(ui.ColorRed, ui.ColorClear, ui.ModifierBold)
+	}
+	height := estimateNumberOfLines(title.Text, termWidth/2)
+	title.SetRect(termWidth/2, currentTop, termWidth, currentTop+height)
+	ui.Render(title)
+	currentTop = currentTop + height + 1
+
+	message := widgets.NewParagraph()
+	message.Text = analysisResult.Message
+	message.Border = false
+	height = estimateNumberOfLines(message.Text, termWidth/2) + 2
+	message.SetRect(termWidth/2, currentTop, termWidth, currentTop+height)
+	ui.Render(message)
+	currentTop = currentTop + height + 1
+
+	if analysisResult.URI != "" {
+		uri := widgets.NewParagraph()
+		uri.Text = fmt.Sprintf("For more information: %s", analysisResult.URI)
+		uri.Border = false
+		height = estimateNumberOfLines(uri.Text, termWidth/2)
+		uri.SetRect(termWidth/2, currentTop, termWidth, currentTop+height)
+		ui.Render(uri)
+		currentTop = currentTop + height + 1
+	}
+}
+
+func estimateNumberOfLines(text string, width int) int {
+	lines := len(text)/width + 1
+	return lines
+}
+
+// func drawGrid(analyzeResults []*analyzerunner.AnalyzeResult) {
```

Comment:
What's this unused block for?

Reply:
oh shoot, nothing. Just need to remove it